.github/workflows/check_release_v0.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v1
```

Comment:
In my opinion, you should unify with master branch, using setup-node@v3
You can see at here
https://github.com/kintone-labs/kintone-ui-component/blob/master/.github/workflows/check_release_v1.yml#L10-L15